src/controllers/inventory.js

```diff
@@ -1,15 +1,21 @@
-import { i19loading, i19loadMore } from '@ecomplus/i18n'
+import { i19loading, i19loadMore, i19sku, i19name, i19price, i19unitsInStock, i19sales } from '@ecomplus/i18n'
 import { $ecomConfig, i18n, price as getPrice } from '@ecomplus/utils'
+import Papa from 'papaparse'
 import ecomClient from '@ecomplus/client'
 
 export default function () {
-  const { $, app, callApi } = window
+  const { $, app, callApi, tabId } = window
 
   const size = 20
   let from = 0
   const datatableOptions = {
     pageLength: size,
-    bLengthChange: false
+    bLengthChange: false,
+    dom: 'Bfrtip',
+    buttons: [
+      'csv', 
+      'excel'
+    ]
```

Comment:
```suggestion
    bLengthChange: false
```

Isso não está alterando nada, acho que você copiou de algum exemplo né...